### ISSUE TITLE
Missing wheel for building myself

### ISSUE THREAD

**Issue by Ausbildungsleiter** [adafruit/circuitpython-build-tools#26]
In README.md at the end is written:

# Building libraries locally

To build libraries built with the build tools you'll need to install the circuitpython-build-tools package.
```
python3 -m venv .env
source .env/bin/activate
pip install circuitpython-build-tools
circuitpython-build-bundles --filename_prefix <output file prefix> --library_location .
```

I had to add after the "source" line

```
pip install wheel
```

**Comment by dhalbert** [comment]
Closed #26 via #27.

**Comment by sommersoft** [comment]
Hopefully this is fixed by #27. 

@Ausbildungsleiter, please give it a day or two for the update to make it to PyPI. Then if you could uninstall `wheel` and try it again.

Thanks for letting us know about this issue!
